### PR TITLE
Updating mangu.hackclub.com MX records

### PR DESCRIPTION
**Pull request by john-kelsey**
custom email

### CODE REVIEW

**Review by Muirrum — CHANGES_REQUESTED** [review]
Hey, can you give more detail on what this PR is supposed to do?

**Comment by john-kelsey** [comment]
Hi , I wanted to configure a custom email with my .hackclub.com domain for
the club , but whatever I did ,did not work , . I would really appreciate
it if you told me how to configure a custom email

File: hackclub.com.yaml [hunk 1]
```diff
@@ -15,9 +15,14 @@
         preference: 10
       - exchange: aspmx3.googlemail.com.
         preference: 10
+      - exchange:  mx1.forwardemail.net
+        preference: 10
+      - exchange:  mx2.forwardemail.net
+        preference: 10
```
Comment (Muirrum) [inline hackclub.com.yaml:21]: Please don't modify the root MX records
```suggestion
```

File: hackclub.com.yaml [hunk 2]
```diff
@@ -15,9 +15,14 @@
...
   - ttl: 5
     type: TXT
     values:
+      - forward-email=johnkelsey.irungu@gmail.com
```
Comment (Muirrum) [inline hackclub.com.yaml:25]:
```suggestion
```

File: hackclub.com.yaml [hunk 3]
```diff
@@ -1618,6 +1623,18 @@ mangu:
...
+    values: 
+      - exchange:  mx1.forwardemail.net
+        preference: 10
+      - exchange:  mx2.forwardemail.net
+        preference: 10
+  - ttl: 5
+    type: TXT
+    values:
+      - forward-email=johnkelsey.irungu@gmail.com
+
```
Comment (Muirrum) [inline hackclub.com.yaml:1637]:
```suggestion
  - ttl: 600
    type: MX
    values: 
      - exchange:  mx1.forwardemail.net.
        preference: 10
      - exchange:  mx2.forwardemail.net.
        preference: 10
  - ttl: 600
    type: TXT
    values:
      - forward-email=johnkelsey.irungu@gmail.com
       
```

This just modifies the TXT record's TTL and adds the required trailing dot to the exchanges

**Review by Muirrum — CHANGES_REQUESTED** [review]

**Pull request closed without merging**